pinot-broker/src/main/java/org/apache/pinot/broker/routing/BrokerRoutingManager.java

```diff
@@ -417,6 +417,12 @@ public synchronized void includeServerToRouting(String instanceId) {
    * Builds/rebuilds the routing for the given table.
    */
   public synchronized void buildRouting(String tableNameWithType) {
```

Comment:
(minor) The argument no longer always have type suffix. Consider putting some comments explaining it

Reply:
Addressed here: https://github.com/apache/pinot/pull/15720/commits/22cc927f77dc967f8207fbb76d0e17012145d098